packages/components-js/package.json

```diff
@@ -31,9 +31,12 @@
-    "replace:fontLatin": "placeholder='<!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_FONT_LATIN-->' && partial=$placeholder$(node -e 'console.log(require(\"@porsche-design-system/components-js/partials\").getFontLinks({ weights: [\"regular\", \"semi-bold\", \"bold\"] }).replace(/https:\\/\\/cdn\\.ui\\.porsche\\.com\\/porsche-design-system/g, \"http://localhost:3001\"))') && regex=$placeholder'.*' && sed -E -e \"s^$regex^$partial^\" public/index.html > public/index.tmp.html && rm public/index.html && mv public/index.tmp.html public/index.html",
+    "replace": "yarn replace:initialStyles && yarn replace:fontLatin && yarn replace:componentChunkLinks && yarn replace:iconLinks",
+    "replace:initialStyles": "placeholder='<!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_INITIAL_STYLES-->' && partial=$placeholder$(node -e 'console.log(require(\"@porsche-design-system/components-js/partials\").getInitialStyles({ prefix: [\"\", \"my-prefix\"] }))') && regex=$placeholder'.*' && sed -E -e \"s^$regex^$partial^\" public/index.html > public/index.tmp.html && yarn replace:renameIndexHtml",
+    "replace:fontLatin": "placeholder='<!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_FONT_LATIN-->' && partial=$placeholder$(node -e 'console.log(require(\"@porsche-design-system/components-js/partials\").getFontLinks({ weights: [\"regular\", \"semi-bold\", \"bold\"] }).replace(/https:\\/\\/cdn\\.ui\\.porsche\\.com\\/porsche-design-system/g, \"http://localhost:3001\"))') && regex=$placeholder'.*' && sed -E -e \"s^$regex^$partial^\" public/index.html > public/index.tmp.html && yarn replace:renameIndexHtml",
+    "replace:componentChunkLinks": "placeholder='<!--PLACEHOLDER_PORSCHE_DESIGN_SYSTEM_COMPONENT_CHUNKS-->' && partial=$placeholder$(node -e 'console.log(require(\"@porsche-design-system/components-js/partials\").getComponentChunkLinks({ components: [\"accordion\", \"banner\", \"button-group\", \"button-pure\", \"button-tile\", \"button\", \"carousel\", \"checkbox-wrapper\", \"content-wrapper\", \"crest\", \"display\", \"divider\", \"fieldset-wrapper\", \"fieldset\", \"flex\", \"flyout\", \"grid\", \"heading\", \"headline\", \"icon\", \"inline-notification\", \"link-pure\", \"link-social\", \"link-tile-model-signature\", \"link-tile\", \"link\", \"marque\", \"modal\", \"model-signature\", \"multi-select\", \"pagination\", \"pin-code\", \"popover\", \"radio-button-wrapper\", \"scroller\", \"segmented-control\", \"select-wrapper\", \"spinner\", \"stepper-horizontal\", \"switch\", \"table\", \"tabs-bar\", \"tabs\", \"tag-dismissible\", \"tag\", \"text-field-wrapper\", \"text-list\", \"text\", \"textarea-wrapper\", \"toast\", \"wordmark\"] }).replace(/https:\\/\\/cdn\\.ui\\.porsche\\.com\\/porsche-design-system/g, \"http://localhost:3001\"))') && regex=$placeholder'.*' && sed -E -e \"s@$regex@$partial@\" public/index.html > public/index.tmp.html && yarn replace:renameIndexHtml",
```

Comment:
I don't think this is very maintainable and easy to forget. I guess we have should create a dedicated node script to take care of this now.